contrib/kube-prometheus/manifests/prometheus-rules.yaml

```diff
@@ -947,7 +947,7 @@ spec:
         message: The configuration of the instances of the Alertmanager cluster `{{$labels.service}}`
           are out of sync.
       expr: |
-        count_values("config_hash", alertmanager_config_hash{job="alertmanager-main"}) BY (service) / ON(service) GROUP_LEFT() label_replace(prometheus_operator_spec_replicas{job="prometheus-operator"}, "service", "alertmanager-$1", "alertmanager", "(.*)") != 1
+        count_values("config_hash", alertmanager_config_hash{job="alertmanager-main"}) BY (service) / ON(service) GROUP_LEFT() label_replace(prometheus_operator_spec_replicas{job="prometheus-operator",controller="alertmanager"}, "service", "alertmanager-$1", "name", "(.*)") != 1
```

Comment:
why do we want `name` here?

Reply:
Because the label is now called `name` instead of `alertmanager` or `prometheus`, because the counter is the same for both controllers:
https://github.com/coreos/prometheus-operator/pull/1984/files#diff-74116c6359432aeaf1431f126493b2acR30

Reply:
ah yes, that's right! :+1: